### ISSUE TITLE
Plugin Presets break core RuneLite settings

### ISSUE THREAD

**Issue by ChipGav** [antero111/plugin-presets#72]
Hi, 

First and foremost, thank you for the plugin. Been using it for ages very happily. 

Unfortunately with the latest update to the game, plugin presets' interaction with the base "Runelite" plugin appears to have broken; you can no longer use plugin presets to resize client sizes. There are weird things happening full stop when using Plugin Presets after the update. Disabling Plugin Presets from modifying the "Runelite" plugin also doesn't work (by unticking Runelite from the list of plugins to be modified).

People have been able to take back control of the size of their window only after uninstalling the Plugin Presets plugin. It's what's being suggested as the current fix on the Discord but I use Plugin Presets multiple times a session and would really appreciate being able to continue using it. 

Thanks!

**Comment by antero111** [comment]
Hi, I saw your discussion on the support channel and I have also noticed that the plugin somehow wasn't able to change the client size setting. It may be the plugins issue but afaik someone said there that core client has something to do with it also... 

The base RuneLite settings support is somewhat manual so it may have been broken since the last update... which has been 2 months ago.

I'll try to debug this tomorrow. Thanks for the report!

**Comment by antero111** [comment]
![NVIDIA_Share_U86cFMZuE1](https://github.com/antero111/plugin-presets/assets/60603897/eddcbedc-08b4-41aa-affe-3f34eae7a6d3)

Such flickering and inconsistensies have not been an issue before the flatlaf update.

**Comment by ChipGav** [comment]
Thanks for your rapid reply! 

Yes, it was working perfectly up until the update only a few days ago. I closed my clients and they were visually different (you'll know more about this than me). I use the plugin several times daily with hotkeys so I will have noticed immediately upon it breaking. Definitely not been broken for a month. 

Appreciate it!

**Comment by ChipGav** [comment]
![ezgif-3-31fd6be779](https://github.com/antero111/plugin-presets/assets/157908014/1b276758-dc49-46e1-9c12-cc6c83e62e32)
This is my testing, in case it's useful. The profiles selected all have different sizes.

**Comment by antero111** [comment]
Should be fixed in v2.8.1. Report new issue if still problem.

**Comment by ChipGav** [comment]
Thanks a ton!
